index.html

```diff
@@ -0,0 +1,154 @@
+            <a class="navbar-brand" href="#">
+              <img src="assets/images/img4.png" alt="navbar logo imag" />
+            </a>
+        </div>
+          <ul class="nav navbar-nav navItem text-secondary" id="navMenu">
+            <li class="active"><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#program">Program</a></li>
+            <li><a href="#sports">Sports</a></li>
+            <li><a href="#">Sponser</a></li>
+            <li><a href="#">Nutrition</a></li>
+          </ul>
+         
+          <div class="humgurger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </nav>
+      </div>
+      <div class="row introduction">
+        
+        <p>"Welcome to Healthline Nutrition!"</p>
+        <h1 class="secondHeader">
+          The Definitive Guide<br />
+          to Healthy Eating in Real Life
+        </h1>
+
+        <p id="second-p">
+          healthy eating doesn’t have to be complicated. It’s entirely possible
+          to nourish your body while enjoying the foods you love.
+        </p>
+        <img src="assets/images/Health-Nutrition.jpg" alt="Nutrition" />
+        
+      </div>
+      <div class="row mainProgram" id="program">
+        <h3>Main Program</h3>
+        <hr />
+        <div class="col-sm card">
+          <img src="assets/images/vegtables.jpg" alt="Vegtables" />
+          <h4 class="card-title">Vegetables</h4>
+          <p class="card-text">
+            Add variety to grilled or steamed vegetables with an herb such as
+            rosemary..
+          </p>
+        </div>
+        <div class="col-sm card">
+          <img src="assets/images/fruits.jpg" alt=" fruits" />
+          <h4 class="card-title">Fruits</h4>
+          <p class="card-text">
+            Fresh, frozen, or canned fruits are great choices. Try fruits beyond apples and bananas such as mango, pineapple or kiwi fruit
+          </p>
+        </div>
+        <div class="col-sm card">
+          <img
+            src="assets/images/calcium.jpg"
+            alt=" calcuim-rich food image"
+          />
+          <h4 class="card-title">calcium-Rich food</h4>
+          <p class="card-text">
+            In addition to fat-free and low-fat milk, consider low-fat and fat-free yogurts without added sugars.
+          </p>
+        </div>
+        <div class="col-sm card">
+          <img src="assets/images/meat.jpg" alt=" meat picture" />
+          <h4 class="card-title">Meat</h4>
+          <p class="card-text">
+            If your favorite recipe calls for frying fish or breaded chicken, try healthier variations by baking or grilling. 
+          </p>
+        </div>
+
+        <p class="text-white" id="wholeProgram">SEE THE WHOLE PROGRAM</p>
+      </div>
+      <div class="row">
+      <section class="sports">
+        <h2>Sports</h2>
+        <hr>
+      </section>
```

Comment:
- [ ] Please work on this section, is missing many elements from the original design, kindly make it as similar as possible to the original design:

|Template|Your design|
|---|---|
|<img width="914" alt="Screen Shot 2022-09-12 at 10 33 37 AM" src="https://user-images.githubusercontent.com/83717285/189702941-55fdc4b7-f025-4957-bffd-93f33f75bece.png">|<img width="1353" alt="Screen Shot 2022-09-12 at 10 33 29 AM" src="https://user-images.githubusercontent.com/83717285/189703028-fa7f8abf-8dc9-489e-a414-282bad8bf8d1.png">|

Reply:
update the dynamic page